### ISSUE TITLE
Does FitNakaRushton.m need correction?

### ISSUE THREAD

**Issue by Chaoyi-1230** [Psychtoolbox-3/Psychtoolbox-3#727]
**Describe the bug**
https://github.com/Psychtoolbox-3/Psychtoolbox-3/blob/master/Psychtoolbox/Psychometric/FitNakaRushton.m

In Line 30 of this file, does the "function" need a corresponding "end" to fit grammar? The original file would encounter an error when included in a package and the compiler raise the error with message "L 30 (C 5-12): ENDCT2: In Line 38 there should be an END to match FUNCTION".

**Comment by DavidBrainard** [comment]
This appears to be a case where the compiler is enforcing different syntax than the interpreter (which does not require the end.

I updated this so that it now has an end, and also switched from implicit function to separate function for the error evaluation (which these days I find to be clearer.)  I added an example in a block comment at the top which demonstrates and tests the function.

To get this before the next release, you will need to download directly from the master at this github repository.

I have not tested with the compiler.  Reopen and let me know if you encounter any further problems
